draft-irtf-cfrg-vdaf.md

```diff
@@ -5526,11 +5526,15 @@ generate and verify multiple, independent proofs. (See {{multiproofs}}.) For
 Prio3, the `PROOFS` parameter controls the number of proofs (at least one) that
 are generated and verified.
 
-In general, Field128 is RECOMMENDED for use in Prio3 when the circuit uses
-joint randomness (`JOINT_RAND_LEN > 0`) and `PROOFS == 1`. Field64 MAY be used
-instead, but `PROOFS` MUST be set to at least `3`. Breaking robustness for
-`PROOFS == 2` is feasible, if impractical; but `PROOFS == 1` is completely
-broken for such a small field.
+For circuits involving joint randomness, we aim for a soundness error of the
+FLP close to `2^-128` in order to mitigate offline attacks. (The soundness
+error is the probability of a random joint randomness accepting an invalid
+measurement plus the base soundness error of the proof system, which is given
+by Theorem 4.3 of {{BBCGGI19}}.) Such circuits MUST use Field128 with at least
+one proof or Field64 with at least three proofs.
+
+In the case of Field64, breaking robustness for two proofs is feasible, if
+impractical; but one proof is completely broken for such a small field.
```

Comment:
This paragraph is just hand-waving as written.  If you want to make an argument for the MUST above, you should provide some explanation for why 3 is the right break point.  Presumably there's something that scales as `2^{k * PROOFS}`, so that `k * PROOFS` is big enough with `PROOFS == 3` but not below?

Reply:
That's close to the truth. I've added the formula that is relevant here. What matters is that, depending on the circuit, Field64/2 might have significantly lower soundness than Field128/1.